driver/bn_driver.hpp

```diff
@@ -317,13 +344,92 @@ std::vector<int> BatchNormDriver<Tgpu, Tref, Tmix>::GetInputTensorLengthsFromCmd
     }
 }
 
+template <typename Tgpu, typename Tref, typename Tmix>
+int BatchNormDriver<Tgpu, Tref, Tmix>::ChkLayout_ShortName()
+{
+    // check for short name of layout type
+    if(inflags.FindShortName("layout") == 'L')
+    {
+        // do noting
+        // found valid short names
+        return 0;
+    }
```

Comment:
Just a nitpick, but surprised this returns an int, and 0 is valid. 
I would expect it to return a bool, and true means valid.

Reply:
Yes! I borrowed the code from conv_driver.hpp and did not bothered to change. I will make this `bool` since that makes more sense here.